### ISSUE TITLE
Fix build

### ISSUE THREAD

**Issue by floscha** [floscha/tinycards-python-api#14]
The unit tests of the build break because of some errors happening while logging in. When adding some sleep time between tests, the chance that all tests pass increases but still does not always work. Figure out where the problem lies and fix accordingly.

**Comment by njam** [comment]
The build currently fails with:
```
Error while trying to log in:
{'message': "Oops, something went wrong! You may be doing something you shouldn't be :)"}
```

Probably because there are no credentials or they are not correct?
The tests succeed for me locally using a test account.

**Comment by floscha** [comment]
For TravisCI, the credentials are already set as environment variables. They are set using the TravisCI web interface since putting them into the *.travis.yml* config is not the smartest way unless you want random strangers to log into your account and mess things up :stuck_out_tongue_winking_eye:

I also noticed that the tests would rather pass in a local environment. Maybe Tinycards blocks the IP of TravisCI or something...

**Comment by njam** [comment]
I see! Since those variables are not available on forks, unfortunately the builds for pull requests will fail.
In #20 I've added credentials for a test account I've just created. This would work also for forks. But of course anyone could log in and delete the account..

**Comment by floscha** [comment]
Finally solved with #24.
